frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/ParametersSettings.tsx

```diff
@@ -0,0 +1,203 @@
+import { useMemo } from "react";
+import { Icon } from "metabase/core/components/Icon";
+import { color } from "metabase/lib/colors";
+import Select, { Option } from "metabase/core/components/Select";
+import { Box, Divider, SegmentedControl, Stack, Text } from "metabase/ui";
+import { ParameterWidget as StaticParameterWidget } from "metabase/parameters/components/ParameterWidget";
+import { getValuePopulatedParameters } from "metabase-lib/parameters/utils/parameter-values";
+
+import type {
+  ActivePreviewPane,
+  EmbedResourceParameter,
+  EmbedResourceType,
+  EmbeddingParameters,
+  EmbeddingParametersValues,
+  EmbeddingDisplayOptions,
+  EmbedResource,
+  EmbedType,
+  EmbedResourceParameterWithValue,
+} from "../EmbedModal.types";
+import EmbedCodePane from "./EmbedCodePane";
+import PreviewPane from "./PreviewPane";
+import {
+  CODE_PREVIEW_CONTROL_OPTIONS,
+  SettingsTabLayout,
+} from "./StaticEmbedSetupPane.styled";
+import { StaticEmbedSetupPaneSettingsContentSection } from "./StaticEmbedSetupPaneSettingsContentSection";
+
+export interface ParametersSettingsProps {
+  activePane: ActivePreviewPane;
+
+  resource: EmbedResource;
+  resourceType: EmbedResourceType;
+  resourceParameters: EmbedResourceParameter[];
+
+  embeddingParams: EmbeddingParameters;
+  previewParameters: EmbedResourceParameter[];
+  parameterValues: EmbeddingParametersValues;
+
+  embedType: EmbedType;
+
+  iframeUrl: string;
+  token: string;
+  siteUrl: string;
+  secretKey: string;
+  params: EmbeddingParameters;
+  displayOptions: EmbeddingDisplayOptions;
+
+  onChangeEmbeddingParameters: (parameters: EmbeddingParameters) => void;
+  onChangeParameterValue: (id: string, value: string) => void;
+
+  onChangePane: (pane: ActivePreviewPane) => void;
+}
+
+export const ParametersSettings = ({
+  activePane,
+  resource,
+  resourceType,
+  resourceParameters,
+  embeddingParams,
+  previewParameters,
+  parameterValues,
+  displayOptions,
+  embedType,
+  iframeUrl,
+  token,
+  siteUrl,
+  secretKey,
+  params,
+  onChangeEmbeddingParameters,
+  onChangeParameterValue,
+  onChangePane,
+}: ParametersSettingsProps): JSX.Element => {
+  const valuePopulatedParameters = useMemo(
+    () =>
+      getValuePopulatedParameters(
+        previewParameters,
+        parameterValues,
+      ) as EmbedResourceParameterWithValue[],
+    [previewParameters, parameterValues],
+  );
+
+  return (
+    <SettingsTabLayout
+      settingsSlot={
+        resourceParameters.length > 0 ? (
+          <>
+            <StaticEmbedSetupPaneSettingsContentSection
+              title={t`Enable or lock parameters`}
+            >
+              <Stack spacing="1rem">
+                <Text>{t`Enabling a parameter lets viewers interact with it. Locking one lets you pass it a value from your app while hiding it from viewers.`}</Text>
+
+                {resourceParameters.map(parameter => (
+                  <div key={parameter.id} className="flex align-center">
```

Comment:
I'm noticing a lot of `div` and `h3` tags and other HTML tags - are these remnants from the old code? Maybe it would be good in another PR to adjust these to mantine while we're here